docs/checks.md

```diff
@@ -0,0 +1,26 @@
+# KubeLinter checks
```

Comment:
So, the table here is currently generated, and this could get out of date. Can we just use the file in `docs/generated/checks.md` for this?